### DISCUSSION TITLE
Cloning encrypted repo on another computer, how-to question

### DISCUSSION THREAD

**Discussion by boelenz** [AGWA/git-crypt#218] [Q&A] [answered]
Hi!

I have encrypted a repository with git-crypt. The master is hosted on a remote computer, so after git commit and git push, I can see that the files are encrypted on this remote master. For encryption I use the symmetrical key.

Now I would like to use this repository on another computer that I own and control. I have tried git clone, copying the key to the other computer and using the key with the git-crypt unlock command, but this did not work, all the files are still encrypted.

My .gitatributes file looks as followes:

```
 filter=git-crypt diff=git-crypt
.gitattributes !filter !diff
```

I would be very happy with a step-by-step short explanation.

**Comment by alerque** [comment, marked as answer]
The first line of your `.gitattributes` file looks off, I think you need a file name pattern. Usually encrypting *everything* in a repository is not a good idea, but if you're sure all the files are secret then at least `*` or something for a pattern should work. Then you need to make sure that it doesn't match `.gitattributes` file. I forget the match priority but you'll want to check that it doesn't encrypt the attributes file too ;-)

After that, your issue is probably with where you are putting "the key". Does `gpg --list-secret-keys` show your private keys correctly? Are you able to use GPG for other things on the new machine?

**Reply by boelenz** [reply]
You are right, I forgot the filespec. I added the * on the first line. The second line, excluding the .gitattibutes file itself, still works.
```
* filter=git-crypt diff=git-crypt
.gitattributes !filter !diff
```
